artifacts/definitions/Windows/Forensics/SRUM.yaml

```diff
@@ -70,13 +81,30 @@ sources:
         WHERE App =~ ExecutableRegex
 
   - name: Network Connections
-    queries:
-    - SELECT AutoIncId as SRUMId,
+    query: |
+        SELECT AutoIncId as SRUMId,
              TimeStamp,
              srum_lookup_id(file=SRUMLocation, accessor=accessor, id=AppId) AS App,
              srum_lookup_id(file=SRUMLocation, accessor=accessor, id=UserId) AS User,
              InterfaceLuid,
              ConnectedTime,
              timestamp(winfiletime=ConnectStartTime) AS StartTime
-      FROM parse_ese(file=SRUMLocation, accessor=accessor, table=NetworkConnectionsGUID)
-      WHERE App =~ ExecutableRegex
+        FROM parse_ese(file=SRUMLocation, accessor=accessor, table=NetworkConnectionsGUID)
+        WHERE App =~ ExecutableRegex
+
+  - name: Network Usage
+    query: |
+        LET SRUMFile <= SELECT FullPath FROM glob(globs=SRUMLocation)
```

Comment:
I wonder why that's needed since the other ones don't need it. Maybe we should have a glob for all of them?

Reply:
So I tried that and then the others failed, really strange behavior since it's the same file. Could it be some sort of race condition?

Reply:
It is possible :-) If you check the requests all the queries are run in series (i.e. one after the other) so it is surprising if they interact.

But in any case maybe it is useful to be able to specify the SRUM location by glob?